jha/src/views/enrolment/MSPEligibilityPage.vue

```diff
@@ -194,14 +196,21 @@ export default {
         return;
       }
       this.saveData();
-      this.navigateToNextPage();
+
+      // Short circuit to form selection if they are ineligible for MSP but still attempting to apply
+      if (this.msgCode === 0 || this.msgCode === 5) {
```

Comment:
Can you update the values here with the constants you created.

Reply:
Done.